app/bot.py

```diff
@@ -1,32 +1,41 @@
-from aiogram import Dispatcher, F
+from aiogram import Dispatcher, F, Bot
 from aiogram.types import BufferedInputFile, Message
 
-from settings import USER_ID
+from settings import *
 from tiktok.api import TikTokAPI
 
 dp = Dispatcher()
+bot = Bot(token=API_TOKEN)
+
 filters = [
     F.text.contains("tiktok.com"),
-    (USER_ID is None) | (F.from_user.id == USER_ID),
+    (not ALLOWED_IDS)
+    | F.chat.id.in_(ALLOWED_IDS)
+    | F.from_user.id.in_(ALLOWED_IDS)
 ]
 
-
 @dp.message(*filters)
 @dp.channel_post(*filters)
 async def handle_tiktok_request(message: Message) -> None:
     entries = [
-        message.text[e.offset : e.offset + e.length]
+        message.text[e.offset: e.offset + e.length]
```

Comment:
Though this is not mentioned anywhere in the project, I'm using [black](https://github.com/psf/black) (as formatter) and [ruff](https://github.com/astral-sh/ruff) (as linter). I probably need to add pyproject.toml with dev dependencies. But anyways, I would appreciate, if you can run them too

Reply:
I ran black formatter before latest commit:

> All done! ✨ 🍰 ✨
> 4 files reformatted, 5 files left unchanged.